context/src/main/java/io/opentelemetry/context/Context.java

```diff
@@ -135,9 +135,33 @@ static Executor taskWrapping(Executor executor) {
    * @since 1.1.0
    */
   static ExecutorService taskWrapping(ExecutorService executorService) {
+    if (executorService instanceof CurrentContextExecutorService) {
+      return executorService;
+    }
     return new CurrentContextExecutorService(executorService);
   }
 
+  /**
+   * Returns an {@link ScheduledExecutorService} which delegates to the provided {@code
+   * executorService}, wrapping all invocations of {@link ExecutorService} methods such as {@link
+   * ExecutorService#execute(Runnable)} or {@link ExecutorService#submit(Runnable)} with the
+   * {@linkplain Context#current() current context} at the time of invocation.
```

Comment:
Can we note that context is not propagated for `scheduleAtFixedRate`, `scheduleWithFixedDelay`?